extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -75,11 +75,9 @@ require([
 		// Ads
 		adTracker.measureTime('adengine.init', 'queue.desktop').track();
 		win.adslots2 = win.adslots2 || [];
-		adEngine.run(adConfigDesktop, win.adslots2, 'queue.desktop');
-
 		scrollHandler.init();
+		adEngine.run(adConfigDesktop, win.adslots2, 'queue.desktop');
```

Comment:
What is the reason we moved `adEngine.run()` after initialization of `scrollHandler`?